src/BridgeFinder.ts

```diff
@@ -169,14 +169,24 @@ export class BridgeFinder extends (EventEmitter as new () => TypedEmitter<Bridge
         }
 
         let bridgeID;
-        const hostname = await this.getHostnameFromIP(ipaddr);
-        logDebug('got hostname from IP:', hostname);
+        let hostname;
+
         try {
+            hostname = await this.getHostnameFromIP(ipaddr);
+            logDebug('got hostname from IP:', hostname);
+
             // If the format of the hub hostname changes, this match can break,
             // and will appear as your credentials not working any longer
             bridgeID = hostname!.match(/[Ll]utron-(?<id>\w+)\.local/)!.groups!.id;
         } catch {
-            bridgeID = ipaddr.replace('.', '_');
+            logDebug("got exception looking up hostname");
```

Comment:
This eats the error. Consider emitting it directly, or saving it up and emitting it in the case of ambiguity (re prior comment)